quickcheck/src/Test/QuickCheck/Gen.ss

```diff
@@ -10,7 +10,7 @@
 
   (define float32ToInt32
     (lambda (f)
-      (let ([buf (scm:foreign-alloc 4)])
```

Comment:
Oops sorry I think I had a syntax error here. A missing closing paren.

Reply:
Tests pass though?? 🤔

Reply:
Well, I tried rerunning CI and it doesn't seem to pick up the wrong changes 🤷